engine/src/main/java/org/terasology/logic/console/ConsoleSystem.java

```diff
@@ -24,22 +25,39 @@
 import org.terasology.input.ButtonState;
 import org.terasology.input.binds.general.ConsoleButton;
 import org.terasology.logic.console.commandSystem.ConsoleCommand;
+import org.terasology.logic.console.ui.MiniChatOverlay;
 import org.terasology.network.ClientComponent;
 import org.terasology.registry.In;
 import org.terasology.rendering.nui.NUIManager;
 
 @RegisterSystem
 public class ConsoleSystem extends BaseComponentSystem {
+
+    private static final ResourceUrn MINICHAT_UI = new ResourceUrn("engine:minichatOverlay");
```

Comment:
I used the MiniChatOverlay to show notifications from the storage service in the menus. If I understand correctly in the menu state there is no `ChatSystem` so I made some modifications to `ConsoleSystem` to show certain notifications in `MiniChatOverlay`. In my opinion it's a good way to notify user when, for example, the synchronization is completed - since it happens asynchronously, I thought that showing message boxes may not be a good idea since they would pop up at random moments possibly annoying the user.
Let me now if something needs to be changed.